utils/io/content/contentreader_test.go

```diff
@@ -30,71 +27,86 @@ type ArrayValue struct {
 	Value string `json:"value"`
 }
 
+func init() {
+	log.SetLogger(log.NewLogger(log.DEBUG, nil))
+}
+
 func TestContentReaderPath(t *testing.T) {
-	searchResultPath := filepath.Join(getTestDataPath(), "contentreaderwriter", searchResult)
-	rr := NewContentReader(searchResultPath, arrayKey)
-	assert.Equal(t, rr.GetFilePath(), searchResultPath)
+	searchResultPath := filepath.Join(getTestDataPath(), searchResult)
+	reader := NewContentReader(searchResultPath, DefaultKey)
```

Comment:
Looks like this reader is not closed.

Reply:
'getTestDataPath()' indicates that the reader points on a file located at the test-data dir and therefore, should not be deleted.